File: Easy/Filter/Math/2652. Sum Multiples.py
"""
https://leetcode.com/problems/sum-multiples/description/
"""


def sumOfMultiples(n: int) -> int:
    ans = 0
    for i in range(1, n + 1):
        if i % 3 == 0 or i % 5 == 0 or i % 7 == 0:
            ans += i
    return ans


n = 7
print(sumOfMultiples(n))

File: PHP/Easy/Filters/Math/2652-Sum_Multiples.php
<?php
// https://leetcode.com/problems/sum-multiples/description/

function sumOfMultiples($n)
{
    $count = 0;
    for ($i = 1; $i <= $n; $i++) {
        if ($i % 3 == 0 or $i % 5 == 0 or $i % 7 == 0) {
            $count += $i;
        }
    }
    return $count;
}